zppy/templates/e3sm_diags.bash

```diff
@@ -178,18 +120,10 @@ create_links_climo_diurnal ${climo_diurnal_dir_source} ${climo_diurnal_dir_ref}
 {%- endif %}
 
 {%- if ("enso_diags" in sets) or ("qbo" in sets) or ("area_mean_time_series" in sets) %}
-{% if run_type == "model_vs_obs" %}
-ts_dir_primary=ts
-{% elif run_type == "model_vs_model" %}
-ts_dir_primary=ts_test
-{%- endif %}
 # Create xml files for time series variables
-ts_dir_source={{ output }}/post/atm/{{ grid }}/ts/monthly/{{ '%dyr' % (ts_num_years) }}
-create_links_ts ${ts_dir_source} ${ts_dir_primary} ${Y1} ${Y2} 5
+ts_dir_primary={{ output }}/post/atm/{{ grid }}/ts/monthly/{{ '%dyr' % (ts_num_years) }}
```

Comment:
2) Rather than using the `create_links` function to populate `ts_dir_primary` with an xml listing files from `ts_dir_source`, we just set `ts_dir_primary` to point the files directly. No more `ts_dir_source` at all. No more file copying or compiling files into a list. Just pointing zppy directly to the source directory.